src/scenes/Activities/scenes/Finalize/styles.js

```diff
@@ -0,0 +1,16 @@
+const styles = () => ({
```

Comment:
that's a good point, I am actually planning to change the component structure and the react-router configuration to make them all inside a parent it has a style.css for it.

Was planning to include that in another PR since I am still trying to get familiar with MobX asap.

Reply:
updated, but will still have an another PR for the rr4 thing.